client/src/features/Campaign/StepDialog.js

```diff
@@ -55,8 +55,10 @@ const StepDialog = (props) => {
   const [editorTemplate, setEditorTemplate] = useState({});
   const [snackbarOpen, setSnackbarOpen] = useState(false);
 
+  const variables = ['name', 'from_first_name'];
+
   useEffect( () => {
-    if(props.step !== undefined && props.open) getTemplate(props.step.templateId);
+    if(props.step !== undefined) getTemplate(props.step.templateId);
```

Comment:
I think it would be neater if you destructure the props, so that you don't need to repeat "props."
Example on line 57:  const { step, delete } = props

Reply:
I also like having props in front lets you know that variable is coming from the props when there's hundred lines of code. idk, I think it's just preference